specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -155,9 +155,9 @@ export function StatsPage(): JSX.Element | null {
   ]);
 
   /* Set Default Layout every time page is rendered*/
-  React.useEffect(() => {
-    setDefaultLayout(defaultLayoutSpec);
-  }, [setDefaultLayout, defaultLayoutSpec]);
+  // React.useEffect(() => {
```

Comment:
Not sure why this was needed so I commented it out. Feel free to re-enable if it's needed